src/locale/locales/de/messages.po

```diff
@@ -1180,113 +1146,106 @@ msgstr "Neue Benachrichtigungen laden"
+msgstr "Vergewissere dich, dass du auch wirklich dorthin gehen willst!"
 
 #: src/view/screens/Profile.tsx:163
 msgid "Media"
 msgstr "Medien"
 
 #: src/view/com/threadgate/WhoCanReply.tsx:139
 msgid "mentioned users"
-msgstr "genannte Nutzer"
+msgstr "erwähnte Benutzer"
 
 #: src/view/com/modals/Threadgate.tsx:93
 msgid "Mentioned users"
-msgstr "Genannte Nutzer"
+msgstr "Erwähnte Benutzer"
 
 #: src/view/screens/Search/Search.tsx:529
 msgid "Menu"
 msgstr "Menü"
 
 #: src/view/com/posts/FeedErrorMessage.tsx:194
 msgid "Message from server"
-msgstr "Meldung vom Server"
+msgstr "Nachricht vom Server"
 
-#: src/view/screens/Moderation.tsx:64
-#: src/view/screens/Settings.tsx:563
-#: src/view/shell/desktop/LeftNav.tsx:395
-#: src/view/shell/Drawer.tsx:509
+#: src/view/screens/Moderation.tsx:64 src/view/screens/Settings.tsx:563
+#: src/view/shell/desktop/LeftNav.tsx:395 src/view/shell/Drawer.tsx:509
 #: src/view/shell/Drawer.tsx:510
 msgid "Moderation"
 msgstr "Moderation"
 
 #: src/view/screens/Moderation.tsx:95
 msgid "Moderation lists"
-msgstr "Moderation Liste"
+msgstr "Moderationslisten"
 
 #: src/view/screens/ModerationModlists.tsx:58
 msgid "Moderation Lists"
-msgstr "Moderation Liste"
+msgstr "Moderationslisten"
 
 #: src/view/shell/desktop/Feeds.tsx:53
 msgid "More feeds"
-msgstr "Mehr Feeds "
+msgstr "Mehr Feeds"
 
 #: src/view/com/profile/ProfileHeader.tsx:548
-#: src/view/screens/ProfileFeed.tsx:360
-#: src/view/screens/ProfileList.tsx:583
+#: src/view/screens/ProfileFeed.tsx:360 src/view/screens/ProfileList.tsx:583
 msgid "More options"
 msgstr "Mehr Optionen"
 
 #: src/view/com/profile/ProfileHeader.tsx:370
 msgid "Mute Account"
-msgstr "Stummes Konto"
+msgstr "Konto stummschalten"
 
 #: src/view/screens/ProfileList.tsx:510
 msgid "Mute accounts"
-msgstr "Stummes Konto"
+msgstr "Konten stummschalten"
 
 #: src/view/screens/ProfileList.tsx:457
 msgid "Mute list"
-msgstr "Stumme Liste"
+msgstr ""
 
 #: src/view/screens/ProfileList.tsx:270
 msgid "Mute these accounts?"
 msgstr "Diese Konten stummschalten?"
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:157
 msgid "Mute thread"
-msgstr "Thema stummschalten"
+msgstr "Thread stummschalten"
 
 #: src/view/screens/Moderation.tsx:109
 msgid "Muted accounts"
-msgstr "Stumme Kontos"
+msgstr "Stummgeschaltete Konten"
 
 #: src/view/screens/ModerationMutedAccounts.tsx:107
 msgid "Muted Accounts"
-msgstr "Stumme Kontos"
+msgstr "Stummgeschaltete Konten"
 
 #: src/view/screens/ModerationMutedAccounts.tsx:115
 msgid "Muted accounts have their posts removed from your feed and from your notifications. Mutes are completely private."
-msgstr "Bei stummgeschalteten Konten werden deren Beiträge aus deinem Feed und deinen Benachrichtigungen entfernt. Stummschaltungen sind vollständig privat."
+msgstr "Bei stummgeschalteten Konten werden ihre Beiträge aus deinem Feed und deinen Benachrichtigungen entfernt. Stummschaltungen sind völlig privat."
```

Comment:
I think we should switch to "geheim" (secret)
```suggestion
msgstr "Die Beiträge stummgeschalteter Konten werden aus deinem Feed und deinen Benachrichtigungen entfernt. Stummschaltungen erfolgen geheim und sind von anderen nicht einsehbar."
```

Reply:
"deren" does fit better in this case than "ihre". Ihre is both 2nd person plural as well as 3rd person plural, deren is only 3rd.